product.html

```diff
@@ -44,50 +44,47 @@
             <h2>Trójmiejski Kalendarz Imprez</h2>
             <h3>Najlepsza wyszukiwarka - najlepsze Imprezy</h3>
 
-            <!--Odtąd dodane opis planowanych funkcji produktu-->
-            <!--modyfikacja: dodać podział na 4 okienka z wyraxnym tytułem i krotkim rozwinięciem.
-            Opis może być rozwijany-->
             <div class="product-container">
                 <hr>
                 <h4>Planowane funkcje produktu</h4>
                 <hr>
 
                 <div class="row">
                     <div class="col-md-6 col-sm-12 col-xs-12 prod_item" data-toggle="collapse" data-target="#speedAndClarity" >
-                        <div><i class="fa fa-eye fa-5x"></i></div>
-                        <div class="product-title" ><h4 class="text-center">Szybkość i czytelność</h4></div>
-                        <p id="speedAndClarity" class="lead text-left product-text collapse">Wyświetlone wydarzenia zawierają krótki opis nazwy, czasu i&nbsp;miejsca, dzięki czemu łatwo i szybko znajdziesz&nbsp;to, co&nbsp;Cię interesuje.
+                        <div class="neon"><i class="fa fa-eye fa-5x"></i>
+                        <h4 class="text-center">Szybkość i czytelność</h4></div>
+                        <p id="speedAndClarity" class="lead text-left product-text collapse">Wyświetlone wydarzenia zawierają krótki opis nazwy, czasu i&nbsp;miejsca, dzięki czemu łatwo i&nbsp;szybko znajdziesz&nbsp;to, co&nbsp;Cię interesuje.
```

Comment:
speedAndClarity - > speed-and-clarity, używamy kebab-case w html, camelCase do JS

Reply:
w kilku miejscach by się przydało zmienić